File: web/src/model/victron_controller/dashboard/from_0_2_0_command_set-charge-battery-extended-mode.ts
// @ts-nocheck
import {SetChargeBatteryExtendedMode as v0_2_0_SetChargeBatteryExtendedMode} from './v0_2_0/Command'
import {SetChargeBatteryExtendedMode as dashboard_SetChargeBatteryExtendedMode} from './Command'

export function convert__command__set_charge_battery_extended_mode__from__0_2_0(from: v0_2_0_SetChargeBatteryExtendedMode): dashboard_SetChargeBatteryExtendedMode {
    return new dashboard_SetChargeBatteryExtendedMode (
        JSON.parse(JSON.stringify(from.value))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_command_set_charge_battery_extended_mode.rs


pub fn convert__command__set_charge_battery_extended_mode__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::command::SetChargeBatteryExtendedMode) -> crate::victron_controller::dashboard::command::SetChargeBatteryExtendedMode {
    crate::victron_controller::dashboard::command::SetChargeBatteryExtendedMode {
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}